### ISSUE TITLE
.popup('reposition')

### ISSUE THREAD

**Issue by joaofig** [vast-engineering/jquery-popup-overlay#89]
I found about this function by reading through the source code, and I have been using it to reposition my tooltips when they overlap the container's limits. By handling the 'onopen' event, I can check whether the tooltip fully fits the view and if not, I can change the offsets and, by calling this function, the tooltip gets repositioned. Just setting the offsets has no effect.

My question is why is this function not documented? As I read in the code, it is being used internally.

Thanks for the great code!

**Comment by embluk** [comment]
Yes, this would be very useful as I found I had the same issues with tool tips becoming out of view within the browser view port. I second this as being useful!

Care to show an example?
